.github/pull_request_template.md

```diff
@@ -0,0 +1,55 @@
+## Title (delete after following this rule)
+
+Your PR title should be prefixed with the Jira ticket ID, e.g. "ADX-123 very important feature".
+
+## Description
+
+Please include a summary of the change(s).
+You should also include motivation: why was this change required and what problem it solves.
+
+Don't forget to reference all other pull requests which are associated to this one below, especially if merging need to be coordinated.
+E.g.
+
+relates fjelltopp/repository_name#issue_number
+
+## Dependency Changes (delete if not applicable)
+
+List any dependencies that have been removed/added/changed in this pull request.
+Has the pipfile.lock or yarn.lock been updated?
+
+## Testing (delete if not applicable)
+
+Please describe the new tests that have been added for this change along with any additional instructions required to reproduce a successful test and/or a list of any relevant details for your test configuration.
```

Comment:
The code should describe itself ideally. However, this section could cover one really important aspect.
1. Why the code lacks unit tests (if applicable)?
2. Why the code lacks integration tests (if applicable)?
3. Why the code lacks any automated tests?

[side note]: If the only way to test something is a manual test we should start collecting them somewhere so we could reproduce the manual tests in the future to avoid regression. This would also allow us to pay off the technical depth in structurized and measurable way.